npm-shrinkwrap.json

```diff
@@ -2735,6 +2777,17 @@
         "ndjson": "^1.5.0",
         "semver": "^7.3.2",
         "tslib": "^2.0.1"
+      },
+      "dependencies": {
+        "balena-errors": {
+          "version": "4.7.1",
```

Comment:
This also bumped balena-errors, so let's also bump it in the package.json so we use the same version across the whole CLI.